packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -286,7 +286,7 @@ class ObjectInspector extends Component<Props> {
         repProps.mode = this.props.mode === MODE.LONG ? MODE.SHORT : MODE.TINY;
       }
       if (expanded) {
```

Comment:
```suggestion
      if (expanded && this.props.mode !== MODE.TINY) {
```

I think this would be better

Reply:
If I change the code as you suggested, nothing change. I printed the `MODE` and I found that they are in `MODE.LONG`. I don't understand why.

Reply:
could you paste a diff of what you have?

Reply:
I've resolved by myself (I think), look at the last commit if that's what you mean.